nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/connect/client/we/PackageListingProvider.java

```diff
@@ -281,12 +283,12 @@ public Object getDetails(@PathParam("pkgId") String pkgId) {
     }
 
     /**
-     * @since 5.6
+     * @since 11.1
```

Comment:
?

Reply:
I've change the prototype of the method so I consider it's a new method, we agreed that we don't have to keep the old methods for these package management related APIs

Reply:
yes, you're right. when the signature changes, it's a new method